Sources/FuturedArchitecture/Architecture/Coordinator.swift

```diff
@@ -23,6 +23,7 @@ public protocol Coordinator: ObservableObject {
     /// Skipping this recommendation may prevent UI updates when changing `@Published` properties, as `rootView` is static.
     /// - Parameter instance: An instance of `Coordinator` which will be retained by the *container*.
     /// - Returns: The container view.
+    @MainActor
     @ViewBuilder
     static func rootView(with instance: Self) -> RootView
```

Comment:
`@ViewBuilder` is always executed on `@MainActor`. This was the case even before the whole `View` protocol got bound to `@MainActor`.
Adding the explicit isolation is necessary to avoid conflicts with `onEvent`.